src/reference/asciidoc/streams.adoc

```diff
@@ -101,29 +119,33 @@ public FactoryBean<StreamsBuilder> myKStreamBuilder(KafkaStreamsConfiguration st
 @Qualifier("&myKStreamBuilder")
 private StreamsBuilderFactoryBean myKStreamBuilderFactoryBean;
 ----
+====
 
-==== JSON Serdes
+==== JSON Serialization and Deserialization
 
-For serializing and deserializing data when reading or writing to topics or state stores in JSON format, Spring Kafka provides a `JsonSerde` implementation using JSON, delegating to the `JsonSerializer` and `JsonDeserializer` described in <<serdes, the serialization/deserialization section>>.
-The `JsonSerde` provides the same configuration options via its constructor (target type and/or `ObjectMapper`).
-In the following example we use the `JsonSerde` to serialize and deserialize the `Foo` payload of a Kafka stream - the `JsonSerde` can be used in a similar fashion wherever an instance is required.
+For serializing and deserializing data when reading or writing to topics or state stores in JSON format, Spring Kafka provides a `JsonSerde` implementation that uses JSON, delegating to the `JsonSerializer` and `JsonDeserializer` described in <<serdes>>.
+The `JsonSerde` implementation provides the same configuration options through its constructor (target type or `ObjectMapper`).
+In the following example, we use the `JsonSerde` to serialize and deserialize the `Cat` payload of a Kafka stream (the `JsonSerde` can be used in a similar fashion wherever an instance is required):
 
+====
 [source,java]
 ----
-stream.through(Serdes.Integer(), new JsonSerde<>(Foo.class), "foos");
+stream.through(Serdes.Integer(), new JsonSerde<>(Cat.class), "cats");
```

Comment:
Yup. My problem with "foo" and "bar" is their history. They come from FUBAR, which stands for Fucked Up Beyond All Repair (or Recognition). I don't want Fucked Up and Beyond All Repair all over the docs and code.

The whole thing was a joke from the same folks who thought the "See Figure 1" joke was funny. Unfortunately, it caught on and spread.

I've picked Thing1, Thing2, Cat, and Hat because I like Dr. Seuss, but I'm game for anything that isn't Fucked Up and Beyond All Repair.

Reply:
Fine with me; but it will be a difficult habit to break moving forward 😄 .

Reply:
My problem with "foo" and "bar" is that they come from FUBAR, which stands for Fucked Up Beyond All Repair (or Recognition). I don't want "Fucked Up" and "Beyond All Repair" all over our docs and code.

I picked Thing1, Thing2, Cat, and Hat, because I like Dr. Seuss and wanted a group of terms that make a set of some sort. I'm fine with anything that isn't Fucked Up and Beyond All Repair.